python/lsst/pipe/tasks/setPrimaryFlags.py

```diff
@@ -36,25 +36,31 @@ class SetPrimaryFlagsConfig(Config):
 class SetPrimaryFlagsTask(Task):
```

Comment:
Please add a nice friendly class docstring per https://developer.lsst.io/python/numpydoc.html?documentation#py-docstring-class-structure

Reply:
I've added this brief class docstring: 
`    """Add isPrimaryKey to a given schema.

    Parameters
    ----------
    schema :
        The input schema.
    isSingleFrame : `bool`
        Flag specifying if task is operating with single frame imaging.
    kwargs:
        Keyword arguments passed to the task.
    """`

Reply:
Thanks! The only missing piece is the type for `schema`. I don't know it offhand... and that's exactly why having it in the docs is useful :)

Reply:
Good catch, thanks! Added (it's `lsst.afw.table.Schema`!)